aziotctl/Cargo.toml

```diff
@@ -3,7 +3,9 @@ name = "aziotctl"
 version = "1.4.0"
 authors = ["Azure IoT Edge Devs"]
 edition = "2021"
-
+homepage = "https://github.com/azure/iot-identity-service"
```

Comment:
I assume it needs this info because it shows up somewhere user-visible, so it would be good for it to be precise and polished. So:

1. ```diff,rust
   -homepage = "https://github.com/azure/iot-identity-service"
   +homepage = "https://azure.github.io/iot-identity-service/"
   ```

1. ```diff,rust
   -description = "aziotctl tool is used for IoT identity service"
   +description = "aziotctl is used to manage the IoT Identity Service and related services."
   ```

1. ```diff,rust
   -description = "The aziotd is the binary for IoT identity service"
   +description = "aziotd is the main binary for the IoT Identity Service and related services."
   ```

1. ```diff,rust
   -description = "aziot-keys"
   +description = "aziot-keys is the default implementation of cryptographic operations used by the Keys Service."
   ```